src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
You don't need a div here, you can style ```:host```

Reply:
done